src/js/modules/customer/customer.view.js

```diff
@@ -68,46 +140,241 @@ export class CustomerView {
     }
   }
 
-  bindOpenModal() {
+  displayEmptyNotification = (message) => {
+    let emptyMessage = createElement('p', 'empty-message')
+    emptyMessage.innerHTML = message
+    this.customersTable.appendChild(emptyMessage)
+    this.previousBtn.classList.add('visibility-hidden')
+    this.nextBtn.classList.add('visibility-hidden')
+  }
+
+  renderGeneralInformation = (customers) => {
+    const activeCustomers = customers.filter(
+      (customer) => customer.status === 'on'
+    )
+
+    this.customersQuantity.innerHTML = customers.length
+    this.activeQuantity.innerHTML = activeCustomers.length
+  }
+
+  renderCustomersTable = (customers, pagination) => {
+    while (this.customersTable.firstChild) {
+      this.customersTable.removeChild(this.customersTable.firstChild)
+    }
+
+    if (!customers.length) {
+      this.displayEmptyNotification(listEmptyMessage)
+      return
+    }
+
+    let _customers = JSON.parse(JSON.stringify(customers))
```

Comment:
You convert object to string then parse to JSON? please recheck, I think it is no need